bot/bot.py

```diff
@@ -70,3 +72,11 @@ async def send_log(self, title: str, details: str = None, *, icon: str = None) -
         embed.set_author(name=title, icon_url=icon)
 
         await devlog.send(embed=embed)
+
+    async def invoke_help_command(self, ctx: commands.Context) -> None:
+        """Invoke the help command or default help command if help extensions is not loaded."""
+        if "bot.exts.core.help" in ctx.bot.extensions:
```

Comment:
Is this what the help command is supposed to look like? This looks like the vanilla.

![image](https://user-images.githubusercontent.com/75038675/204685114-a95e9c23-c24a-402f-9b04-e2f54d4a9c57.png)

Reply:
I'd rather have it merged than worry about the help text for now

Reply:
I misunderstood what the help invoke function is about. It works (ex: &help aoc x)